service/handlers.go

```diff
@@ -177,6 +177,14 @@ func handlePoolPost(c *Core, w *ResponseWriter, r *Request) {
 		w.Error(err)
 		return
 	}
+
+	b, err := json.Marshal(api.EventPool{
+		PoolID: pool.ID.String(),
+	})
+	if err != nil {
+		w.Error(err)
+	}
+	c.publishEvent("pool-new", b)
```

Comment:
Maybe call `json.Marshal` inside `publishEvent`?
```suggestion
	if err := c.publishEvent("pool-new", api.EventPool{PoolID: pool.ID.String()}); err != nil {
		w.Error(err)
		return
	}
```